### ISSUE TITLE
Could not load file or assembly WebEye.Stream.Win32

### ISSUE THREAD

**Issue by sekulicb** [jacobbo/WebEye#79]
Hi.

I have Lenovo MIIX 3 8 inch tablet with Windows 10 32 bit installed. When the application is running, clicking on *Add Local Stream* causes exception with the message:

Could not load file or assembly 'WebEye.Stream.Win32.dll' or one of its dependencies. The specified module could not be found.

One thing to mention. The project is built from a master branch on a different machine and then copied over to the tablet. If I try to build the solution directly on the tablet I get following build error: 

Resource file "StreamControl.resx" cannot be found.

Running the WebEye.StreamContro.Wpf.Demo gets me to the exception above (on the tablet). Running on the desktop PC with USB web camera shows no problems. Let me know if you need more information.

Thanks in advance

**Comment by jacobbo** [comment]
Could not load file or assembly 'WebEye.Stream.Win32.dll' or one of its
dependencies. The specified module could not be found.
Hi, do you see that file on the tablet?
One of dependencies is missing. Can you run depends utility and check what
exactly is missing?

**Comment by sekulicb** [comment]
Yes, that is correct. What type of dependency utility app? Do you have any recommendations?

**Comment by jacobbo** [comment]
This one
http://www.dependencywalker.com

**Comment by sekulicb** [comment]
Is there a specific result that I could copy-paste here?

**Comment by sekulicb** [comment]
![0-02-0a-451027482062d0cd444249ca951cbc7f046cfd4d3ff2e265b9613a1fb08a7d6b_83be7ae](https://user-images.githubusercontent.com/20758693/76439818-453fcf80-63bd-11ea-9d8a-3cb187d8a402.jpg)

**Comment by jacobbo** [comment]
You want to check WebEye.Stream.Win32.dll file. The picture should look like this
![image](https://user-images.githubusercontent.com/1826752/76474926-2bba7a00-63f5-11ea-8614-98b55bd5b275.png)

**Comment by sekulicb** [comment]
DLL inserted
![1](https://user-images.githubusercontent.com/20758693/76612389-86022a80-651c-11ea-97e7-6eee070721d8.jpg)

1.
![2](https://user-images.githubusercontent.com/20758693/76612392-869ac100-651c-11ea-9c88-40205245648e.jpg)

2.
![3](https://user-images.githubusercontent.com/20758693/76612387-84d0fd80-651c-11ea-9cf9-0dbb90ffe3c0.jpg)

**Comment by jacobbo** [comment]
Visual C++ Redistributable for Visual Studio 2015 is missing, you can
download it here:
https://www.microsoft.com/en-us/download/details.aspx?id=48145

**Comment by sekulicb** [comment]
Hi. This actually solved one problem but got two more 😄 

Now that I can actually select the camera from the combo box and start it, the video is delayed by a few seconds. What I mean by that is when I make a move or gesture, the video is not lagging, it's just not keeping up with me, it just doesn't record in real-time.

The second problem is when I switch from back to the front camera or vice versa. I get a message in the far-right textbox - Failed: avformat_open_input() failed: I/O

Thanks

**Comment by jacobbo** [comment]
OK, thanks for your feedback, I'll have a look.

**Comment by thedude61636** [comment]
@jacobbo 
Have you been able to track the issue with avformat_open_input()?
I'm also getting that issue on a client's device, he's using a microsoft lifecam if that makes a difference.
